### DISCUSSION TITLE
Support GLB?

### DISCUSSION THREAD

**Discussion by StarsTesla** [mitsuba-renderer/mitsuba3#1438] [Feature requests]
I wonder, why only support obj and ply?

**Comment by njroussel** [comment]
Hi @StarsTesla 

I don't think that there is any specific reason why this was never implemented, but rather that we never had a need for it. Usually, by using other tools, we'd always manage to convert to ply/obj.

This would be a nice self-contained plugin if someone were to implement it.

**Comment by StarsTesla** [comment]
@njroussel In my view, the bsdf material need to be manual setting like xml or python dict, instead of automatically import their material or texture map, this is kinda waste of time espeically for complex 3d model and its materia for rendering.
